src/main/java/com/koczy/kurek/mizera/thesisbrowser/hibUtils/HibernateUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.koczy.kurek.mizera.thesisbrowser.hibUtils;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static final SessionFactory sessionFactory;
+
+    static {
+        Configuration conf = new Configuration();
+        conf.configure();
+        try {
+            sessionFactory = conf.buildSessionFactory();
+        } catch (Exception e) {
+            System.err.println("Initial SessionFactory creation failed." + e);
+            throw new ExceptionInInitializerError(e);
```

Comment:
Jak to poleci to jest potem gdzieś łapane ?

Reply:
To jest Error, nie Exception więc jego się nie łapie. Jak wystąpi to oznacza, że nie udało się postawić appikacji. Chyba tak powinno to wyglądać.